templates/base/store/global/reducer.js

```diff
@@ -0,0 +1,25 @@
+import { APP_ERROR, APP_LOADED } from './actions';
+
+// @Initial State
+export const initialState = {
+    loading: true,
+    error: false
+}
+
+const globalReducer = (state = initialState, action) => {
+    switch(action.type) {
+        case APP_ERROR:
+            return {
```

Comment:
Use the payload from action.payload, no point hard coding an error state to always true